cli/index.ts

```diff
@@ -176,6 +186,12 @@ async function serve(port: number, pathToSpec: string, options: Options = {}) {
         'Content-Type': 'text/html',
       });
     } else if (request.url === '/spec.json') {
+      if (options.watch && isUrl) {
+        spec = await loadAndBundleSpec(pathToSpec).catch(e => {
```

Comment:
You don't need `.catch` when you do `await`. Any error will be thrown and can be catched by try/catch.